### PR TITLE
D8 nid 515

### PR DESCRIPTION
**Pull request by omahm**

### CODE REVIEW

File: migrate_nidirect_taxo/config/install/migrate_plus.migration.upgrade_d7_taxonomy_term_recipe_course_types.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
+uuid: 00810d2c-1c88-4ba0-8304-1da7c66d9325
+langcode: en
+status: true
+dependencies: {  }
+id: upgrade_d7_taxonomy_term_recipe_course_types
```
Comment (johangant) [inline migrate_nidirect_taxo/config/install/migrate_plus.migration.upgrade_d7_taxonomy_term_recipe_course_types.yml]: Shouldn't this be removed as recipes are all coming out too?

File: migrate_nidirect_taxo/config/install/migrate_plus.migration.upgrade_d7_taxonomy_term_recipe_ingredient.yml [hunk 2]
```diff
@@ -0,0 +1,76 @@
+uuid: b9ddd7ef-8c70-4ac7-842b-dfa349b996c7
+langcode: en
+status: true
+dependencies: {  }
+id: upgrade_d7_taxonomy_term_recipe_ingredient
```
Comment (johangant) [inline migrate_nidirect_taxo/config/install/migrate_plus.migration.upgrade_d7_taxonomy_term_recipe_ingredient.yml]: Remove file as recipes are being pulled out as well?

File: migrate_nidirect_taxo/config/install/migrate_plus.migration.upgrade_d7_taxonomy_term_recipe_special_diet.yml [hunk 3]
```diff
@@ -0,0 +1,76 @@
+uuid: 22f8fe71-3bb9-4bfe-b66b-1d2c3942ceb3
+langcode: en
+status: true
+dependencies: {  }
+id: upgrade_d7_taxonomy_term_recipe_special_diet
```
Comment (johangant) [inline migrate_nidirect_taxo/config/install/migrate_plus.migration.upgrade_d7_taxonomy_term_recipe_special_diet.yml]: More recipe migration config to remove?

File: migrate_nidirect_utils/src/Command/NidirectMigratePreCommand.php [hunk 4]
```diff
@@ -199,4 +199,40 @@ protected function task_remove_recipe_nodes() {
...
+  // phpcs:disable
+  protected function task_remove_umbrella_body_node_metatags() {
+  // phpcs:enable
+    $this->drupal7DatabaseQuery("DELETE FROM metatag INNER JOIN node ON node.nid = metatag.entity_id WHERE node.type = 'nidirect_ub'");
+  }
+
+  /**
+   * Remove umbrella body node revisions.
+   */
+  // phpcs:disable
```
Comment (johangant) [inline migrate_nidirect_utils/src/Command/NidirectMigratePreCommand.php]: +1 for selective use of this. phpcs can be a bit fussy with Drupal console commands

File: migrate_nidirect_taxo/config/install/migrate_plus.migration.upgrade_d7_taxonomy_term_recipe_course_types.yml (hunk 1 above)
Comment (omahm) [reply migrate_nidirect_taxo/config/install/migrate_plus.migration.upgrade_d7_taxonomy_term_recipe_course_types.yml]: Yeah that's me jumping the gun a little on 515 before 514 was merged in

**Pull request merged**